src/index.html

```diff
@@ -11,6 +11,17 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#site" class="form form--01"
+     method="post">
+      <input type="text" class="form__textbox form--01__textbox"
+       placeholder="Try &quot;Los Angeles&quot;"
+      data-qa="big" >
+    </form>
+
+    <form action="#site" class="form form--02"
```

Comment:
in this case GET will be correct, also you should use uppercase when it comes to method attribute